src/index.js

```diff
@@ -17,19 +17,24 @@ export default class Scrl {
             onTick: () => {},
             friction: .7, // 1 - .3
             acceleration: .04,
+            element: window,
         }
-
+        
         // merge options
         this.options = {
             ...defaults,
             ...options,
         };
 
+        if( !this._elementIsWindow() && !(this.options.element instanceof Element) ) {
+            console.warn('[srcl] options.element expects either window or an HTMLElement. Given value:', options.element);
```

Comment:
I think we can just exit here for invalid input. It's not like the rest of the things would work anyway. 

```suggestion
            throw new Error(`[srcl] options.element expects either window or an HTMLElement. Given value: ${options.element}`);
```

Reply:
Added the `throw`